### DISCUSSION TITLE
Different Between PairLogitPairwise and PairLogit and Impact on Categorical Values

### DISCUSSION THREAD

**Discussion by ManRev** [catboost/catboost#2624] [General]
I have binary variables only and use two models:

pool = Pool(X, y, cat_features=cat_features, group_id=group_id)
model1 = CatBoostRanker(eval_metric="**PairLogit**", pool... ) and model2 = CatBoostRanker(eval_metric="**PairLogitPairwise**", pool... )

- model1 provides a tree that looks normal (a node is for instance [Xi = 0]). 
- model2, however, behaves very weirdly. A node is, for instance [{Xi} counter_type=Counter prior_numerator=0, value>10].

It seems that model2 turns all the variables into numerical variables. But even that doesn't make any sense: the value of Xi is 0 or 1, why is the node displaying e.g., value>10?

Could you explain me what is happening?

**Comment by ek-ak** [comment]
Hello!
[Xi = 0] in you tree  means that that Xi was used as one hot feature with value 0
[{Xi} counter_type=Counter prior_numerator=0, value>10] is description for Counter: https://catboost.ai/en/docs/concepts/algorithm-main-stages_cat-to-numberic
